module/VuFind/src/VuFind/RecordDriver/WorldCat.php

```diff
@@ -64,11 +64,6 @@ public function setRawData($data)
             $data = $data['fullrecord'];
         }
 
-        // Make sure the XML has an appropriate header:
-        if (strlen($data) > 2 && substr($data, 0, 2) != '<?') {
```

Comment:
Is this being removed as unnecessary, or was it actively breaking something?

Reply:
MarcXml serialization class already takes care of it so there is no need to do it here.